config/locales/event_types/en.yml

```diff
@@ -9,3 +9,4 @@ en:
     email_changed: Email address changed
     phone_changed: Phone number changed
     phone_confirmed: Phone confirmed
+    timestamp: '%{date} at %{time} (Eastern)'
```

Comment:
This key is very specific to eastern time, can we put that in the key name somewhere? `timestamp_eastern` ?